spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -57,6 +57,8 @@ Therefore, these alternative approaches are strongly preferred for newly-written
 
 A Jakarta Data provider is required to support the Query by Method Name extension in Jakarta Data 1.0.
 
+NOTE: A Jakarta Data provider backed by a key-value or wide-column datastore is not required to support Query by Method Name.
```

Comment:
This is probably more a question for @otaviojava , but this seems wrong. Surely these database types must be capable of at least some subset of it, otherwise how can they be supported at all in Jakarta Data?

Reply:
Well that's what we have in the table right now. Every action is marked "Not required" for these kinds of databases. It seemed strange to me too.

Reply:
The `key-value` can only support any operation related to the key, with some exceptions. Otherwise, it will return an `UnsupportedOperationException`; the same generally happens with most wide-columns, such as Hbase, ScyllaDB, and Cassandra. 

This message was sent because all possible key operations were converted at the BasicRepository. Therefore, we decided to proceed in this direction.

⚠️ Cassandra and ScyllaDB can support if including the secondary index, but, it it not like

Reply:
> all possible key operations were converted at the BasicRepository. Therefore, we decided to proceed in this direction.

It's probably worth pointing out that since then a number of operations have been removed from `BasicRepository`, including some that were id-based, so that might not be true anymore.  In any case I would say it seems wrong because Jakarta Data does not force the use of `BasicRepository`.

Reply:
OK, so then we need to sort out what are the real requirements here.

I'm not sure that such discussions should block this PR, since that's a longer conversation, which will be facilitated by having the current requirements presented in a better/clearer way.

Reply:
OTOH, I guess I would say that if key-value databases only support key-based operations, then what is the value in having QbMN for those databases? They can already do everything possible with `@Delete` and `@Find`.

Reply:
That is my opinion as well.
It does not make sense to key-value database this feature.